src/components/tx/ExecutionMethod/index.tsx

```diff
@@ -0,0 +1,122 @@
+import type { ReactNode } from 'react'
+import {
+  Box,
+  Divider,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Paper,
+  Radio,
+  RadioGroup,
+  Skeleton,
+  Stack,
+  SvgIcon,
+  Typography,
+} from '@mui/material'
+import RelayerIcon from '@/public/images/common/relayer.svg'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import WalletIcon from '@/components/common/WalletIcon'
+import { type ConnectedWallet } from '@/hooks/wallets/useOnboard'
+import css from './styles.module.css'
+import useRemainingRelays from '@/hooks/useRemainingRelays'
+
+enum ExecutionType {
+  RELAYER = 'Via relayer',
+  CONNECTED_WALLET = 'With connected wallet',
+}
+
+const RelayingChip = ({ children }: { children: ReactNode }) => <Box className={css.relayingChip}>{children}</Box>
+
+const RelayerLabelContent = ({ value, remaining }: { value: string; remaining: number | undefined }) => (
+  <>
+    <SvgIcon component={RelayerIcon} inheritViewBox />
+    <Typography fontWeight="bold">{value}</Typography>
+    {remaining !== undefined ? (
+      <RelayingChip>
+        <SvgIcon component={InfoIcon} fontSize="small" />
+        {/* Will the relays limit be variable? Can we fetch it from BE? */}
```

Comment:
Yes, the endpoint is `/v1/relays/:chainId/:address`. It returns:

```
{
  remaining: number,
  expiresAt: number | undefined
}
```

Reply:
The comment is wrt the max relays per hour. I was thinking if it could also be changed over time.